build/yamls/flow-aggregator/base/conf/flow-aggregator.conf

```diff
@@ -8,5 +8,10 @@
 # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
 #flowExportInterval: 60s
 
-# Provide the transport protocol for the flow aggregator collecting process, which is tcp or udp.
-#aggregatorTransportProtocol: "tcp"
+# Provide the transport protocol for the flow aggregator collecting process, which is tls, tcp or udp.
+# Default to "tls"
```

Comment:
remove, we do not explicitly specify what the default is since it is implied by the commented out line (see the other config parameters in this file)